code/sorting/bubble_sort/BubbleSort.java

```diff
@@ -0,0 +1,31 @@
+package bubblesort;
```

Comment:
Remove this line. This will cause error for many as they may not have the same package.